hack/lib/init.sh

```diff
@@ -20,13 +20,22 @@ source "${ROOT_DIR}/hack/lib/util.sh"
 : "${DELETE_PVCS:=true}"
 
 : "${DEPLOY_PLATFORM:=openshift}"
```

Comment:
Should probably rename this to OCP_DEPLOY_PLATFORM and then just set DEPLOY_PLATFORM to it `: ${DEPLOY_PLATFORM:=$OCP_DEPLOY_PLATFORM}`